app/src/common/shared/com/igalia/wolvic/ui/widgets/WindowWidget.java

```diff
@@ -539,6 +539,10 @@ int getSelectedPanel() {
         return mLibrary.getSelectedPanelType();
     }
 
+    public void hideAllPanel() {
+        hideAllPanel(true);
+    }
+
```

Comment:
Again likely better to have this integrated in the commit that does the hideX -> hideAll transition